tap_sftp/discover.py

```diff
@@ -14,15 +14,19 @@ def discover_streams(config):
 
     tables = config['tables']
     for table_spec in tables:
+        original_name = table_spec['table_name'].replace("/", "_")
+        visible_name = table_spec['table_name']
+        if original_name != visible_name:
+            LOGGER.info(f"Table name has been renamed from {original_name} to {visible_name} due to unsupported characters.")
```

Comment:
@keyn4 This log is backwards, you renamed from `visible_name` to `original_name` right? I think your variable names here are confusing lol

Reply:
🫠 fixed the variable names it was supposed to be the other way.